app/models/miq_server/configuration_management.rb

```diff
@@ -2,16 +2,11 @@ module MiqServer::ConfigurationManagement
   extend ActiveSupport::Concern
   include ConfigurationManagementMixin
 
-  def get_config(type = "vmdb")
-    if is_local?
-      VMDB::Config.new(type)
-    else
-      VMDB::Config.for_resource(type, self)
-    end
+  def config
```

Comment:
As much as rubocop doesn't like `get_*` methods I think just `config` might be a bit too likely to conflict. Plus it's not very descriptive. Maybe something like `server_config` or `server_settings`?

Reply:
This way my train of thought exactly when rubocop complained. But after I checked the actual usage, `config` did not seem that bad anymore, since most of the time, this is called as `MiqServer.my_server.config` and the rest of the calls being in the form of `MiqServer.find(...).config`. I think that having `MiqServer.my_server.server_config` or `MiqServer.find(...).server_settings` does not add much clarity (and that duplicated server in the case of `my_server` just looks bad) over bare `config`.

If my argument is not strong enough, I will replace `config`/`set_config` with `server_settings`/`set_server_settings`.